src/components/Hero/Hero.scss

```diff
@@ -130,11 +130,31 @@
   margin: 0 auto;
   opacity: 0.65;
   padding: 6px;
-  pointer-events: none;
   position: absolute;
   top: 92%;
   transform: translate(-50%, -50%);
 
+  @keyframes bounce {
+    0%,
+    20%,
+    50%,
+    80%,
+    100% {
+      transform: translateY(0);
+      transform: translate(-50%, 0);
+    }
+
+    40% {
+      transform: translateY(-30px);
+      transform: translate(-50%, -7px);
+    }
+
+    60% {
+      transform: translateY(-15px);
+      transform: translate(-50%, -3.5px);
+    }
+  }
```

Comment:
```suggestion
 @keyframes bounce {
    0%,
    20%,
    50%,
    80%,
    100% {
      transform: translate(-50%, 0);
    }

    40% {
      transform: translate(-50%, -7px);
    }

    60% {
      transform: translate(-50%, -3.5px);
    }
  }
```

Reply:
Sorry, I didn´t delete it from a previous test